test/unit/dataSourceProvider.js

```diff
@@ -103,6 +104,8 @@ describe('Provide in Sources', function () {
     context.elevationLayers = [elevationlayer];
     context.colorLayers = [colorlayer];
 
+    stub.restore();
```

Comment:
I move everything in a before. We need to mock the fetching of json and image as we insteancate many layers: PlanarLayer, colorLayer, ElevationLayer and GeometryLayer.
But later on we will need to mock these (te creation of Layer) instead of mocking the fetching used in the layer creation.